CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Postman UVM Changelog
 
+#### v1.7.5 (October 5, 2018)
+* Updated to chai assertion library and removed Expect.js 
```

Comment:
Use `-` for consistency. Also, this message needs to be changed.
Something like: `Migrated tests to chai expect assertions`